packages/types/src/checkin.ts

```diff
@@ -1,3 +1,17 @@
+interface CrontabSchedule {
+  type: 'crontab';
+  // The crontab schedule string, e.g. 0 * * * *.
+  value: string;
+}
+
+interface IntervalSchedule {
+  type: 'interval';
+  value: number;
+  unit: 'year' | 'month' | 'week' | 'day' | 'hour' | 'minute';
+}
+
+type MonitorSchedule = CrontabSchedule | IntervalSchedule;
+
 // https://develop.sentry.dev/sdk/check-ins/
 export interface CheckIn {
```

Comment:
wondering why we have snake case in this type. I guess it saves us logic and size not having to map.

Reply:
yeah I don't want to write a `toJSON` to similar, but we might end up adopting it if needed (we do have `Session` vs. `SerializedSession` type for ex.)